test/integration/WebProxy.spec.ts

```diff
@@ -11,7 +12,7 @@ describe('WebProxy', () => {
     config = {
       webproxy: {
         disable: false,
-        port: 8080,
+        port: await getUnusedPort(),
```

Comment:
I'd rather have this value be assigned to a var in the beginning of the file/test, in case more usages will pop out.